empire/server/modules/powershell/privesc/powerup/service_exe_stager.py

```diff
@@ -21,6 +21,7 @@ def generate(
     ):
 
         # read in the common module source code
+        module.script_path = "privesc/PowerUp.ps1"
```

Comment:
This should go in the `sevice_exe_stager.yaml` instead of being set in code. See [here](https://github.com/BC-SECURITY/Empire/blob/master/empire/server/modules/powershell/credentials/credential_injection.yaml#L56) as an example.